azurerm/resource_arm_app_service.go

```diff
@@ -226,6 +236,25 @@ func resourceArmAppService() *schema.Resource {
 			// https://github.com/Azure/azure-rest-api-specs/issues/1697
 			"tags": tagsForceNewSchema(),
 
+			"site_credential": {
+				Type:     schema.TypeList,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"user_name": {
```

Comment:
we'd tend to write this as `username` - I'll push a commit to make this consistent

Reply:
Thanks for the updating.